ast-visual-studio-extension/CxPreferences/CxPreferencesUI.cs

```diff
@@ -37,7 +39,14 @@ public void Initialize(CxPreferencesModule preferencesModule)
             cxPreferencesModule = preferencesModule;
             tbApiKey.Text = cxPreferencesModule.ApiKey;
             tbAdditionalParameters.Text = cxPreferencesModule.AdditionalParameters;
+            ascaCheckBox.Checked = cxPreferencesModule.AscaCheckBox;
+            UpdateAscaUiState();
         }
+        private void UpdateAscaUiState()
```

Comment:
unnecessary  function, delete it and write this line instead

Reply:
The UpdateAscaUiState() function is actually used in two places: in Initialize and in AscaCheckBox_CheckedChanged